apps/nextjs-website/src/components/atoms/AlertPart/AlertPart.tsx

```diff
@@ -17,7 +17,7 @@ const AlertPart = ({ alertStyle, severity, text, title }: AlertPartProps) => {
       severity={severity}
       sx={{ borderRadius: '4px', marginBottom: spacing(5), ...alertStyle }}
     >
-      <AlertTitle>{title}</AlertTitle>
+      {title && <AlertTitle>{title}</AlertTitle>}
```

Comment:
```suggestion
      <AlertTitle>{title}</AlertTitle>
```
title is no longer optional